src/vehicle.h

```diff
@@ -164,6 +164,9 @@ struct vehicle_part : public JsonSerializer, public JsonDeserializer
     /** Try to set fault returning false if specified fault cannot occur with this item */
     bool fault_set( const fault_id &f );
 
+    /** Get wheel diameter times wheel width (inches^2) or return 0 if part is not wheel */
```

Comment:
Easier to write, search for, display in exotic fonts (say, dedicated roguelike ones).
It's more convenient, like space-less paths even though everything supports spaces in paths.